.sync.yaml

```diff
@@ -0,0 +1,6 @@
+:global:
```

Comment:
I'd have done this separately to make the reviewers' jobs easier, but in this case it's probably fine.